main.cpp

```diff
@@ -261,201 +298,206 @@ struct StereoWidenerAudioPlugin
-
-    // link to manufacturer website
-    char website(); //return is actionable by user
-
-    // not be copied
+    char displayTextBody();
+    char website();
     void copyProtection();
 };
-/*
 
-Thing 7) Company
+char License::displayTextBody()
+    {
+    return 't';
+    }
+char License::website()
+    {
+    return 'w';
+    }
+void License::copyProtection(){}
 
-*/
 struct Company
 {
-
-    //name (std::string)
     std::string companyName = "SBL Plugins";
-    //location (std::string)
     std::string companyLocation = "Atlanta";
-    //number of employees (int)
     int companyEmployees = 1;
-    //amount of income per year (float)
     float companyIncome = 0.0f;
-    //cost of office equipment (float)
     float equipmentCost = 200.25f;
 
-    //3 things it can do:
-    // create plugins
-    int createPlugin(); //return is related to function parameter
-
-    // set authorization methods
+    int createPlugin();
     void authEnable();
-
-    // sign legal agreements
     void signContract();
 };
-/*
 
-Thing 8) Signal Processor
+int Company::createPlugin()
+    {
+    return 1;
+    }
+void Company::authEnable(){}
+void Company::signContract(){}
+
 
-*/
 struct SignalProcessor
 {
-
-    //samples (float)
     float numSamples = 256.0f;
-    //channels (int)
     int numChannels = 2;
-    //type (std::string)
     std::string type = "Unknown";
-    //buffer (float)
     float numBuffer = 2.2f;
-    //program (int)
     int program = 4;
 
-    //3 things it can do:
-    // receive audio
     void receiveAudio();
-
-    // convert raw audio level to decibels
-    float convertToDecibels(); //return is related to function parameter
-
-    // send audio
+    float convertToDecibels();
     void sendAudio();
 };
-/*
 
-Thing 9) DSP Engine
+void SignalProcessor::receiveAudio(){}
+float SignalProcessor::convertToDecibels()
+    {
+    return 0;
+    }
+void SignalProcessor::sendAudio(){}
+
 
-*/
 struct DSPEngine
 {
-
-    //value of incoming gain (double)
-    double gainInputLevel = 0;
-    //duration of gain ramp (double)
-    double gainRampDuration = 3482;
-    //amount of effect (float)
+    float gainInputLevel = 0.0f;
+    float gainRampDuration = 3482.0f;
     float wetLevel = 33.1f;
-    //name of effect (std::string)
     std::string effect = ("Chorus" "Phase" "Flange");
-    //value of output level (float)
     float gainOutputLevel = -6.4f;
 
-    // 3 things it can do:
-    // create a chorus effect
     void createChorus();
-
-    // modify gain level
-    float modifyGain(); //return is related to function parameter
-
-    // disable output
-    bool outputEnabled(); //return is related to function parameter
+    float modifyGain();
+    bool outputEnabled();
 };
-/*
 
-Thing 10) Equalizer Audio Plugin
+void DSPEngine::createChorus(){}
+float DSPEngine::modifyGain()
+    {
+    return (gainInputLevel + gainOutputLevel);
+    }
+bool DSPEngine::outputEnabled()
+    {
+    return true;
+    }
```

Comment:
Fix this formatting
`outputEnabled` is describing a state.  it is not describing an action.  pick a better function name that describes an action.
- [ ] revise

Reply:
Done